client/src/fluid/FluidView.ml

```diff
@@ -25,7 +26,11 @@ let viewCopyButton tlid value : msg Html.html =
     ; ViewUtils.eventNoPropagation
         "click"
         ~key:("copylivevalue-" ^ value ^ TLID.toString tlid)
-        (fun m -> ClipboardCopyLivevalue (value, m.mePos)) ]
+        (fun m ->
```

Comment:
Doing the escape here instead of inside `ClipboardCopyLivevalue` because of its usage in [ViewSidebar](https://github.com/darklang/dark/blob/main/client/src/canvas/ViewSidebar.ml#L615) to intentionally set `quotes` for hash.

Reply:
I don't see a great reason for ViewSidebar to do that, but also I would say the ViewSidebar use case is odd as it reuses something clearly meant for copying the live value. I think it would be better to move this to the ClipboardCopyLiveValue handler instead and ignore the ViewSidebar thing.

Reply:
Good to know. Move the changes to `ClipboardCopyLivevalue` handler now.

Reply:
Good to know. Move the changes to `ClipboardCopyLivevalue` handler now.

Reply:
Good to know. Move the changes to `ClipboardCopyLivevalue` handler now.